src/util/strings.ts

```diff
@@ -50,6 +50,39 @@ export const getIncrementRatingButtonLabel = (
 	}'s ${attribute} from ${interimRating} to ${increment ? nextRatingAbove : nextRatingBelow}`;
 };
 
+/**
+ * Returns the string needed for the props in the PopUp component in the Submit Rating page
+ * @param attribute
+ * @param propsType
+ * @returns
+ */
+export const getPopUpProps = (
+	attribute: string,
+	propsType:
+		| "buttonAccessibility"
+		| "text"
+		| "modalAccessibility"
+		| "closeButtonAccessibilityLabel"
+): string => {
+	let displayString!: string;
+
+	switch (propsType) {
+		case "buttonAccessibility":
+			displayString = `A button that triggers a pop up to open. This will explain what ${attribute} is.`;
```

Comment:
less techy more consumer-oriented. this isn't a description, it should be concise and clear and CTA-ish.

ex: Press this button to find out more about the ${attribute} rating.